app/test/e2e/v2/get-area.spec.js

```diff
@@ -103,9 +103,17 @@ describe('Get areas - V2', () => {
             [{ userId: USERS.USER.id }, { userId: USERS.MANAGER.id }, { userId: USERS.ADMIN.id }]
         );
 
-        const response = await requester.get(`/api/v2/area?loggedUser=${JSON.stringify(USERS.ADMIN)}&all=true`);
+        // First sync areas
+        const response = await requester.post(`/api/v2/area/sync`).send({ loggedUser: USERS.ADMIN });
```

Comment:
this is basically a single test for two endpoints - not sure I like it.

Reply:
The sync call needs to be here to simulate a real use case (which is calling sync + get with all=true filter). This is a test for the get endpoint with all=true, and there's a separate test suite for the sync endpoint, so not testing both endpoints here :)